active/SSTI.py

```diff
@@ -0,0 +1,32 @@
+"""
+github : https://github.com/knassar702/scant3r
+--
+* Coded by : Khaled Nassar @knassar702
+* Version : 0.5#Beta
+
+"""
+def scanNode(sas, msg):
+  pass
+
+ssti_payloads = {
+    'abcd{{6*6}}123':'abcd36123',
+    'abcd${6*6}123':'abcd36123',
+    'abcd<% 6*6 %>123':'abcd36123'
+    }
+
+def scan(sas, msg, param, value):
+  for p,M in ssti_payloads.items():
```

Comment:
Think it's worth changing this to `payload,evidence` throughout in case others might want to use this as a template or example in the future?